xml/System.Buffers/ReadOnlySequence`1.xml

```diff
@@ -463,9 +463,9 @@
       </Parameters>
       <Docs>
         <param name="position">The <see cref="T:System.SequencePosition" /> of which to get the offset.</param>
-        <summary>Returns the offset of a <paramref name="position" /> within this sequence from the start.</summary>
-        <returns>The offset from the start of the sequence.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns the offset of a <paramref name="position" /> within this sequence.</summary>
+        <returns>The offset in the sequence.</returns>
+        <remarks>The returned offset is not a zero-based index from the start. Subtract <code>mySequence.GetOffset(mySequence.Start)</code> from the returned offset to obtain the zero-based index offset.</remarks>
```

Comment:
Is this remark specifically for .NET 10+? If so, we should retain both remarks and annotated them with versions.